src/NightwatchServiceProvider.php

```diff
@@ -88,9 +88,10 @@ final class NightwatchServiceProvider extends ServiceProvider
     /**
      * @var array{
      *     enabled?: bool,
-     *     sampling: array{
-     *        requests: float,
-     *        commands: float,
+     *     sampling?: array{
+     *        requests?: float,
+     *        commands?: float,
```

Comment:
Improving the types here to better reflect the state people may have config files in.